app/helpers/view_helper.rb

```diff
@@ -22,4 +22,17 @@ def bat_contact_email_address
   def bat_contact_mail_to(name = nil, subject: nil, link_class: "govuk-link")
     mail_to bat_contact_email_address, name || bat_contact_email_address, subject: subject, class: link_class
   end
+
+  def sort_options
```

Comment:
I think this would make more sense in the `ResultsView` as that already handles the view logic where this is used and it seems a bit strange referencing it from here. It ties this helper to only being used on the `results#index` anyway.

Reply:
Ok good call